sdks/python/apache_beam/dataframe/schemas.py

```diff
@@ -305,7 +305,7 @@ def _dtype_to_fieldtype(dtype):
   elif dtype.kind == 'S':
     return bytes
   else:
-    raise TypeError("Unsupported dtype in proxy: '%s'" % dtype)
+    return Any
```

Comment:
Note the execution time unbatching code "just works" because it always iterates over the Series' values. The only thing blocking this was finding an appropriate type mapping.